test/msw-api/setup-server/scenarios/on-unhandled-request/error.test.ts

```diff
@@ -17,7 +17,13 @@ afterAll(() => server.close())
 test('errors on unhandled request when using the "error" value', async () => {
   const getResponse = () => fetch('https://test.mswjs.io')
 
-  await expect(getResponse()).rejects.toThrow(
-    'request to https://test.mswjs.io/ failed, reason: [MSW] Error: captured a GET https://test.mswjs.io/ request without a corresponding request handler.',
-  )
+  await expect(getResponse()).rejects.toThrowErrorMatchingInlineSnapshot(`
```

Comment:
Can we replace snapshot with asserting an exact message? I don't think snapshot is needed here, as it adds an extra complexity to the test.